python/rpdk/java/resolver.py

```diff
@@ -1,27 +1,27 @@
 from rpdk.core.jsonutils.resolver import UNDEFINED, ContainerType
 
 PRIMITIVE_TYPES = {
-    "string": "String",
-    "integer": "Integer",
-    "boolean": "Boolean",
-    "number": "Double",
-    UNDEFINED: "Object",
+    "string": {"default": "String"},
+    "integer": {"default": "Integer", "int32": "Integer", "int64": "Long"},
+    "boolean": {"default": "Boolean"},
+    "number": {"default": "Double"},
+    UNDEFINED: {"default": "Object"},
 }
 
 
 def translate_type(resolved_type):
     if resolved_type.container == ContainerType.MODEL:
         return resolved_type.type
     if resolved_type.container == ContainerType.PRIMITIVE:
-        return PRIMITIVE_TYPES[resolved_type.type]
+        return PRIMITIVE_TYPES[resolved_type.type].get(resolved_type.type_format)
```

Comment:
what if `type_format` is not in the `dict` then we get `None`

Reply:
good call. will handle